src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  let marriedStatus = <p className="Person__partner">I am not married</p>;
+
+  if (isMarried) {
+    if (sex === 'm') {
+      marriedStatus = (
+        <p className="Person__partner">{partnerName} is my wife</p>
+      );
+    } else {
+      marriedStatus = (
+        <p className="Person__partner">{partnerName} is my husband</p>
+      );
+    }
+  }
```

Comment:
```suggestion
  const partner = sex === 'm' ? 'wife' : 'husband';
  const marriedStatus = isMarried ? `${partnerName} is my ${partner}` : 'I am not married';
  
```
enough to make just two ternary and decide which text will be rendered in 
`<p className="Person__partner">{marriedStatus}</p>`